zsh/functions/tat

```diff
@@ -30,8 +30,9 @@ create_if_needed_and_attach() {
 }
 
 tat() {
-  path_name="$(basename "$PWD" | tr . -)"
-  session_name=${1-$path_name}
+  parent="$(basename $(dirname "$PWD"))"
+  current="$(basename "$PWD" | tr . -)"
```

Comment:
They're not always projects, so I went with `child` in dccd86a57ad7a1a5533c8afe240c91f5fc575ef3 :+1: